tests/test_cli.py

```diff
@@ -23,6 +23,7 @@ def wrap(*args, **kwds):
     return wrap
 
 
+@with_default_env
```

Comment:
Does this change relate to the content of this PR?

Reply:
yes, tests were failing without that, due to my local cli set to dev env